makefiles/scan-build.inc.mk

```diff
@@ -7,6 +7,7 @@ SCANBUILD_ENV_VARS := \
   BINDIR \
   BINDIRBASE \
   BOARD \
+  BUILDOUT_DIR \
```

Comment:
I do not know how to test the `scan-build` feature.

Reply:
I added it here bc BINDIR and BINBASE and others are listed, too. Better remove, for now?